PureBasicIDE/ToolbarManagement.pb

```diff
@@ -562,6 +562,7 @@ DataSection
   Data.l #MENU_FindNext:            Data$ "Menu:FindNext"
   Data.l #MENU_FindPrevious:        Data$ "Menu:FindPrevious"
   Data.l #MENU_FindInFiles:         Data$ "Menu:FindInFiles"
+  Data.l #MENU_Replace:             Data$ "Menu:Replace"
```

Comment:
You need to update the count in `#NB_ToolbarMenuItems` at the top of this file for the newly added entry.

Sorry, didn't see that when I first reviewed this PR.

Reply:
Well seen, my fault, I hadn't looked on how it was loaded.

Reply:
No worries. It is an easy mistake to make. Maybe this should be changed to be more like the other `DataSections` which have no predefined count but are terminated by a special value instead. Makes it easier to add new values like this.